server/src/test/java/org/elasticsearch/index/query/ExistsQueryBuilderTests.java

```diff
@@ -62,12 +62,12 @@ protected void doAssertLuceneQuery(ExistsQueryBuilder queryBuilder, Query query,
         String fieldPattern = queryBuilder.fieldName();
         Collection<String> fields = context.simpleMatchToIndexNames(fieldPattern);
         Collection<String> mappedFields = fields.stream().filter((field) -> context.getObjectMapper(field) != null
-                || context.getMapperService().fieldType(field) != null).collect(Collectors.toList());
+            || context.getMapperService().fieldType(field) != null).collect(Collectors.toList());
         if (fields.size() == 1 && mappedFields.size() == 0) {
             assertThat(query, instanceOf(MatchNoDocsQuery.class));
             MatchNoDocsQuery matchNoDocsQuery = (MatchNoDocsQuery) query;
             assertThat(matchNoDocsQuery.toString(null),
-                    containsString("No field \"" + fields.iterator().next() + "\" exists in mappings."));
+                containsString("User requested \"match_none\" query."));
```

Comment:
Let's remove this assertion on the error message. I suspect that attempting to maintain this error message is not going to be worth the efforts, and there is no point in asserting on an error message that is not helpful.

Reply:
Sure, done.